lib/modules/platform/gitlab/index.ts

```diff
@@ -1340,3 +1347,37 @@ export async function filterUnavailableUsers(
   }
   return filteredUsers;
 }
+
+export async function expandGroupMembers(
+  reviewersOrAssignees: string[]
+): Promise<string[]> {
+  const expandedReviewersOrAssignees: string[] = [];
+  const normalizedReviewersOrAssigneesWithoutEmails: string[] = [];
+
+  // Skip passing user emails to Gitlab API, but include them in the final result
+  for (const reviewerOrAssignee of reviewersOrAssignees) {
+    if (reviewerOrAssignee.indexOf('@') > 0) {
+      expandedReviewersOrAssignees.push(reviewerOrAssignee);
+      continue;
+    }
+
+    // Normalize the potential group names before passing to Gitlab API
+    normalizedReviewersOrAssigneesWithoutEmails.push(
+      noLeadingAtSymbol(reviewerOrAssignee)
+    );
+  }
+
+  for (const reviewerOrAssignee of normalizedReviewersOrAssigneesWithoutEmails) {
+    try {
+      const members = await getMemberUsernames(reviewerOrAssignee);
+      expandedReviewersOrAssignees.push(...members);
+    } catch (err) {
+      // Unable to fetch group or is not a group, so add back to list.
+      logger.debug(
```

Comment:
do we get a 404 if the group isn't found? so we log the error if it's not 404?

Reply:
we do get a 404 if the group is not found, so whether that is the reason (or its auth related, etc.), we debug log for all those cases.

Reply:
ok, we should silently ignore 404 or at least don't log the http error